packages/p2p/src/stores/general-store.js

```diff
@@ -46,6 +46,11 @@ export default class GeneralStore extends BaseStore {
         return this.props?.client || {};
     }
 
+    @computed
+    get setCurrentFocus() {
```

Comment:
```suggestion
    get setCurrentFocus() {
```
Correct me if i'm wrong but this sets focus for the input correct? Why are we handling it here? `store` should be used for data that will be used in multiple places. For focus the better idea is to useRef and do it on a component level

Reply:
Will take a look at this code again